sormas-app/app/src/main/java/de/symeda/sormas/app/backend/common/DatabaseHelper.java

```diff
@@ -1460,6 +1463,7 @@ public void onUpgrade(SQLiteDatabase db, ConnectionSource connectionSource, int
 
 				getDao(Event.class).executeRaw("ALTER TABLE events RENAME TO tmp_events;");
 
+				//@formatter:off
```

Comment:
There are A LOT of formatting changes in this file that shouldn't be in here; doesn't look like the formatter that you added are doing their job as well (some of them are not typed correctly btw.); can you please revert that?